src/config/buttons.js

```diff
@@ -0,0 +1,7 @@
+export const labelsById = {
+  'watch-video': 'Watch Video',
```

Comment:
can we make `watch-video` & `learn-more` id's generic like the others? if we're saying the client can add whatever text they want we should make these buttons flexible. when two buttons are next to each other, the common term is `primary-button` or `primary-cta` and `secondary-button`/`cta` or `ghost-button` (when it has that outline). there could be a future world where the client wants to show case something else in the header area.. eg partner site, podcast, blog, pr article, etc

Reply:
Absolutely, good suggestion.